graylog2-server/src/main/java/org/graylog2/lookup/Catalog.java

```diff
@@ -54,37 +63,24 @@ public Entry load(String id) {
                         var catalog = contentPackService.getEntityExcerpts();
                         var excerpt = catalog.get(id);
                         if (excerpt != null) {
-                            return new Entry(excerpt.type().name(), excerpt.title());
+                            if (excerpt.title() != null) {
+                                return new Entry(excerpt.title());
+                            } else {
+                                return new Entry(id);
+                            }
                         } else {
-                            return new Entry("Unknown entity: " + id, "Unknown entity: " + id);
+                            return Entry.UNKNOWN_ENTITY;
                         }
                     }
                 });
     }
 
-    public String getTitle(final GRN grn) {
+    public Entry getEntry(final GRN grn) {
         try {
-            var item = cache.get(grn.entity());
-            if(item.title() != null) {
-                return item.title();
-            } else {
-                return "Unknown entity: " + grn;
-            }
+            return cache.get(grn.entity());
         } catch (ExecutionException cex) {
-            return "Unknown entity: " + grn;
+            return Entry.UNKNOWN_ENTITY;
```

Comment:
I am not sure if we should just catch and ignore the exception here. I think we should let it bubble up to the caller and handle it there.